searx/settings.yml

```diff
@@ -1,6 +1,14 @@
 general:
     debug : False # Debug mode, only for development
     instance_name : "searx" # displayed name
+    git_url: https://github.com/searx/searx
+    git_branch: master
+    issue_url: https://github.com/searx/searx/issues
+    docs_url: https://searx.github.io/searx
+    public_instances: https://searx.space
+    contact_url: False # mailto:contact@example.com
```

Comment:
Is is possible to:
* keep `contact_url` is the `general` section : this is a value most admin will probably modify.
* move the other new keys to a `brand` section just after the `general` section. These values won't be modified expect for a new searx brand.

Reply:
> Is is possible to:

Yes, I can move such options to new brand section.

Reply:
I will pick your 1074ad77 / thanks!